migrations/20240218134105_proposals.js

```diff
@@ -0,0 +1,47 @@
+exports.up = async function (knex, Promise) {
+  await knex.schema.table('posts', table => {
+    table.bigInteger('quorum')
+    table.text('proposal_status').index()
+    table.text('proposal_outcome').index()
+    table.text('proposal_type')
+    /*
+      added proposal_ prefix to status and outcome because they are very generic column names
+      and I don't want to confuse future developers as to why they are largely null
+    */
+    table.text('anonymity')
```

Comment:
wouldnt this be boolean?

Reply:
I was leaving space for general anonymity:
- anonymous posts (and thus anonymous proposals)
- anonymous votes
- both

Reply:
i would make those separate fields if and when we need that. thats why i would call it anonymous_voting